libslide/src/parser.rs

```diff
@@ -7,12 +7,13 @@ mod expression_pattern_parser;
 pub use expression_parser::parse as parse_expression;
 pub use expression_pattern_parser::parse as parse_expression_pattern;
 
+use crate::common::Span;
+use crate::diagnostics::Diagnostic;
 use crate::grammar::*;
-use crate::scanner::types::{Token, TokenType};
+use crate::scanner::types::{Token, TokenType as TT};
 use crate::utils::PeekIter;
```

Comment:
can you replace this as TokenType::* and that way you don't need to write tt every time

Reply:
No because the token types overlap with grammar names

Reply:
should we fix that or do you think not worth

Reply:
we can fix it but let’s do it in a separate pr